dashboard/app/assets/javascripts/admin.js.erb

```diff
@@ -0,0 +1,26 @@
+/**
+ * Dynamic generation and event bindings for project admin section of the admin box
+ */
+
+if (!window.dashboard) {
+  window.dashboard = {};
+}
```

Comment:
I think you should be able to omit this since https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/javascripts/namespace.js gets required before everything else in `application.js.erb`.